buildpack/telemetry/fluentbit.py

```diff
@@ -0,0 +1,131 @@
+import logging
+import os
+import subprocess
+import shutil
+import socket
+
+import backoff
+
+from buildpack import util
+from buildpack.telemetry import splunk
+
+
+NAMESPACE = "fluentbit"
+CONF_FILENAME = f"{NAMESPACE}.conf"
+FILTER_FILENAME = "redaction.lua"
+FLUENTBIT_ENV_VARS = {
+    "FLUENTBIT_LOGS_PORT": os.getenv("FLUENTBIT_LOGS_PORT", default="5170"),
+}
+
+
+def _set_default_env(m2ee):
+    for var_name, value in FLUENTBIT_ENV_VARS.items():
+        util.upsert_custom_environment_variable(m2ee, var_name, value)
+
+
+def stage(buildpack_dir, destination_path, cache_path):
+
+    if not is_fluentbit_enabled():
+        return
+
+    logging.info("Fluent Bit is being installed...")
```

Comment:
Let's add a log line after the fluent bit is successfully installed. We don't need it here.

Reply:
Changed